## The impact of deep learning in treating disease and developing new treatments

*There will be some overlap with the Categorize section, and we may have to
determine which methods categorize individuals and which more directly match
patients with treatments.  The sub-section titles are merely placeholders.*

### Categorizing patients for clinical decision making

*How can deep learning match patients with clinical trails, therapies, or
other interventions?  As an example, [@doi:10.1016/j.jalz.2015.01.010]
predicts individuals who are most likely to decline during a clinical trial
and benefit from the treatment.*

### Effects of drugs on transcriptomic responses

*We discussed a few papers that operate on Library of Network-Based Cellular
Signatures (LINCS) gene expression data.  We could briefly introduce the
goals of that resource and comment on the deep learning applications.  In the
Issues, we had reservations about whether the improvements in expression
prediction are good enough to make a practical difference in the domain and
feature selection and construction.*

### Ligand-Based Prediction of Bioactivity

*Deep learning has been applied in some fashion to structure-based,
compound-protein interaction-based, and ligand-based prediction problems
with the overall goal of finding chemical compounds that impair protein
activity.  AtomNet [@arxiv:1510.02855] is worth including as an example
of how 3D convolutions can be used for structure-based modeling.  The main
emphasis should be on ligand-based approaches.  There was substantial hype
after the Merck Kaggle competition, which we can comment on.  Multitask
networks have been impactful here.  There are also creative approaches
for unsupervised and supervised learning with chemical compounds.  Neural
networks have opened new avenues for representation learning.  These
approaches may not be dominant on supervised learning
tasks yet, but they are uniquely tailored to neural networks and have
great future potential.*

- Short introduction to problem, related reviews
- Briefly mention compound-protein interaction-based and structure-based
approaches because those will not be the main focus
- Introduce ligand-based approaches, early success on Merck Kaggle,
standard supervised learning with fingerprints or descriptors as features
- Multi-task networks and conclusions from these studies
- Representation learning and novel recent approaches including graph
convolutions, autoencoders, and one shot learning
- Possibly highlight DeepChem and other efforts on the software and
reusability side
- State of the field, "transformative" assessment

### Modeling Metabolism and Chemical Reactivity

Add a reveiw here of metabolism and chemical reactivity.
